monai/bundle/scripts.py

```diff
@@ -171,6 +172,10 @@ def _get_ngc_bundle_url(model_name: str, version: str) -> str:
     return f"https://api.ngc.nvidia.com/v2/models/nvidia/monaitoolkit/{model_name.lower()}/versions/{version}/zip"
 
 
+def _get_nvstaging_bundle_url(model_name: str, version: str, repo: str) -> str:
```

Comment:
`_get_ngc_private_bundle_url()`

Reply:
Thanks, updated it in the latest commit.